docs/Catalog/Preferences.md

```diff
@@ -77,4 +86,12 @@ ui:
         - $.key4[0]
 ```
 
-![](../imgs/package-list-selective-metadata.png)
+![Example of package_description use](../imgs/package-list-selective-metadata.png)
+
+#### `ui.athena` example
```

Comment:
Can you help me understand what's going on here? Is this telling users how to hardcode the default 'workflow' that shows up on the Athena pane?

Reply:
Yes, I missed it when tinkered with `git add --patch`